### ISSUE TITLE
image generation quality

### ISSUE THREAD

**Issue by xLuge** [CUC-MIPG/Edit-Transfer#1]
![Image](https://github.com/user-attachments/assets/b2850255-df59-4221-b266-a585ee8af9af)hi, nice work, but , the image reconstruction quality is very low, What details need to be noted during inference?

**Comment by Looperswag** [comment]
The input for inference should be a 4x4 grid with 1024x1024 resolution?

**Comment by DreamLoveBetty** [comment]
The input for inference should be a 4x4 grid with 1024x1024 resolution?

**Comment by Orannue** [comment]
Hi @Looperswag @DreamLoveBetty , thank you for your attention！ If you use our provided model weights, we highly recommend using a 4×4 grid with a 1024×1024 resolution as input, since our dataset follows this format.

We have not tested data in other formats yet. However, you can try using inputs with different resolutions in a 4×4 grid and modify the mask resolution in the following code:
https://github.com/CUC-MIPG/Edit-Transfer/blob/c44a846da1cbf7fb5ed6b9938299b3f9fff645a3/edit_transfer.py#L27 
Alternatively, you can construct a new dataset in a different format and test it.

Let us know if you have any questions!

**Comment by Looperswag** [comment]
> Hi [@Looperswag](https://github.com/Looperswag) [@DreamLoveBetty](https://github.com/DreamLoveBetty) , thank you for your attention！ If you use our provided model weights, we highly recommend using a 4×4 grid with a 1024×1024 resolution as input, since our dataset follows this format.
> 
> We have not tested data in other formats yet. However, you can try using inputs with different resolutions in a 4×4 grid and modify the mask resolution in the following code:
> 
> [Edit-Transfer/edit_transfer.py](https://github.com/CUC-MIPG/Edit-Transfer/blob/c44a846da1cbf7fb5ed6b9938299b3f9fff645a3/edit_transfer.py#L27)
> 
> Line 27 in [c44a846](/CUC-MIPG/Edit-Transfer/commit/c44a846da1cbf7fb5ed6b9938299b3f9fff645a3)
> 
>  mask=torch.zeros(1,1024,1024) 
> 
> Alternatively, you can construct a new dataset in a different format and test it.
> Let us know if you have any questions!

Is there any quick way to construct a 4*4 grid given images?

**Comment by Orannue** [comment]
> > Hi [@Looperswag](https://github.com/Looperswag) [@DreamLoveBetty](https://github.com/DreamLoveBetty) , thank you for your attention！ If you use our provided model weights, we highly recommend using a 4×4 grid with a 1024×1024 resolution as input, since our dataset follows this format.
> > We have not tested data in other formats yet. However, you can try using inputs with different resolutions in a 4×4 grid and modify the mask resolution in the following code:
> > [Edit-Transfer/edit_transfer.py](https://github.com/CUC-MIPG/Edit-Transfer/blob/c44a846da1cbf7fb5ed6b9938299b3f9fff645a3/edit_transfer.py#L27)
> > Line 27 in [c44a846](/CUC-MIPG/Edit-Transfer/commit/c44a846da1cbf7fb5ed6b9938299b3f9fff645a3)
> > mask=torch.zeros(1,1024,1024)
> > Alternatively, you can construct a new dataset in a different format and test it.
> > Let us know if you have any questions!
> 
> Is there any quick way to construct a 4*4 grid given images?

The quickest approach is using FLUX to directly generate a 4×4 grid image, but the results are often unsatisfactory.
So we generate a 1×2 grid first and then manually select two results to compose a 4×4 grid. Alternatively, we use off-the-shelf editing methods to modify images and construct the final 4×4 grid.
